packages/firebase_ml_custom/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
 ## 0.0.1
```

Comment:
Since we must republish please update the 0.0.1 description and add a 0.0.2 description for a new version that we can publish